frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/review-and-create-step/review-and-create-step.tsx

```diff
@@ -79,50 +85,63 @@ export const ReviewAndCreate: React.FC<ReviewAndCreateProps> = ({ state }) => {
           </ListItem>
         )}
       </ReviewItem>
-      <ReviewItem title={t('ceph-storage-plugin~Capacity and nodes')}>
-        <ListItem>
-          {t('ceph-storage-plugin~Cluster capacity: {{capacity}}', {
-            capacity: formattedCapacity,
-          })}
-        </ListItem>
-        <ListItem>
-          {t('ceph-storage-plugin~Selected nodes: {{nodeCount, number}} node', {
-            nodeCount: nodes.length,
-            count: nodes.length,
-          })}
-        </ListItem>
-        <ListItem>
-          {t('ceph-storage-plugin~CPU and memory: {{cpu, number}} CPU and {{memory}} memory', {
-            cpu: totalCpu,
-            memory: humanizeBinaryBytes(totalMemory).string,
-          })}
-        </ListItem>
-        <ListItem>
-          {t('ceph-storage-plugin~Zone: {{zoneCount, number}} zone', {
-            zoneCount: zones.size,
-            count: zones.size,
-          })}
-        </ListItem>
-      </ReviewItem>
-      <ReviewItem title={t('ceph-storage-plugin~Security and network')}>
-        <ListItem>
-          {t('ceph-storage-plugin~Encryption: {{encryptionStatus}}', { encryptionStatus })}
-        </ListItem>
-        {hasEncryption && (
+      {!isMCG && (
+        <ReviewItem title={t('ceph-storage-plugin~Capacity and nodes')}>
           <ListItem>
-            {t('ceph-storage-plugin~External key management service: {{name}}', {
-              name: encryption.advanced ? kms.name.value : t('ceph-storage-plugin~Disabled'),
+            {t('ceph-storage-plugin~Cluster capacity: {{capacity}}', {
+              capacity: formattedCapacity,
             })}
           </ListItem>
-        )}
-        {isMultusSupported && (
           <ListItem>
-            {t('ceph-storage-plugin~Network: {{networkType}}', {
-              networkType: NetworkTypeLabels[networkType],
+            {t('ceph-storage-plugin~Selected nodes: {{nodeCount, number}} node', {
+              nodeCount: nodes.length,
+              count: nodes.length,
             })}
           </ListItem>
-        )}
-      </ReviewItem>
+          <ListItem>
+            {t('ceph-storage-plugin~CPU and memory: {{cpu, number}} CPU and {{memory}} memory', {
+              cpu: totalCpu,
+              memory: humanizeBinaryBytes(totalMemory).string,
+            })}
+          </ListItem>
+          <ListItem>
+            {t('ceph-storage-plugin~Zone: {{zoneCount, number}} zone', {
+              zoneCount: zones.size,
+              count: zones.size,
+            })}
+          </ListItem>
+        </ReviewItem>
+      )}
+      {isMCG ? (
+        <ReviewItem title={t('ceph-storage-plugin~Security')}>
+          <ListItem>{t('ceph-storage-plugin~Encryption: Enabled')}</ListItem>
```

Comment:
If encryption is always `Enabled` for MCG shouldn't the "Enable Encryption" checkbox in previous "Security" step should always be checked+disabled as well ?? (User can further decide whether to check KMS setting or not).
Wdyt @afreen23  ??

Reply:
yes it should be. I guess I missed that.

Reply:
![Screenshot from 2021-08-24 17-26-22](https://user-images.githubusercontent.com/25664409/130612349-aaf64c8c-be01-48c8-a5c8-f6a9406a57eb.png)